### ISSUE TITLE
Add Montserrat

### ISSUE THREAD

**Issue by bryceerobertson** [anki-geo/ultimate-geography#640]
Is there a reason Montserrat isn't included in the deck?

**Comment by aplaice** [comment]
Yes:

https://github.com/anki-geo/ultimate-geography/blob/master/CONTRIBUTING.md#political-geography

(There's links to long discussions there, as well...)

(It's a sufficiently small dependent territory (population ≈ 4000), that we removed it as not interesting enough for the normal user.)

You can get it back with the temporary "legacy" deck:

https://github.com/anki-geo/ultimate-geography/releases/tag/v4.0

or in "Hardcore Geography":

https://github.com/anki-geo/hardcore-geography

(not yet released).

**Comment by bryceerobertson** [comment]
Thanks for letting me know :)

For me, the deck isn't about what's interesting but rather about what exists. So I'd be strongly in favour of including places like Montserrat. Are there any other similar places that are excluded?

**Comment by aplaice** [comment]
> Are there any other similar places that are excluded?

Yes, everything in the first half of [this file](https://github.com/anki-geo/hardcore-geography/blob/main/src/data/country.csv)

(The second half contains places which are included in the main deck, but without capital and flag.)

**Comment by bryceerobertson** [comment]
Ah ok thanks. I'm too much of a noob on GitHub to know how to download those cards so I'll probably just make cards for them myself.

**Comment by axelboc** [comment]
Following #661, is it maybe worth adding a disclaimer in the README to explain why some content may appear to be "missing" and to point people to https://github.com/anki-geo/hardcore-geography?
